### PR TITLE
Update translation command to make i18n.

### PR DESCRIPTION
**Pull request by Faakhir30**
References plone/volto/pull/6274

Updated translation command at https://6.docs.plone.org/i18n-l10n/contributing-translations.html#translate-volto  to use make i18n.

### CODE REVIEW

File: docs/i18n-l10n/resync-translations.md [hunk 1]
```diff
@@ -103,10 +103,10 @@ When the release manager requests to create a new `plone.app.locales` release, t
 ## Resync translations in Volto
 
 In Volto, the GitHub test setup warns a developer when their new contributions require regenerating the translation file.
-This is done by running a yarn script as follows:
+This is done by running a make script as follows:
```
Comment (stevepiercy) [inline docs/i18n-l10n/resync-translations.md]:
```suggestion
This is done by running a make command as follows:
```

File: docs/i18n-l10n/resync-translations.md [hunk 2]
```diff
@@ -116,9 +116,9 @@ This will update the PO files and will leave them ready to be translated by tran
 
 ## Create a release of Volto with the new translations
 
-The Volto release process requires running the same yarn script as in the previous step.
+The Volto release process requires running the same make script as in the previous step.
```
Comment (stevepiercy) [inline docs/i18n-l10n/resync-translations.md]:
```suggestion
The Volto release process requires running the same make command as in the previous step.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Original term was not correct, but fixed here.

**Comment by Faakhir30** [comment]
@stevepiercy I see you have already fixed the broken redirect link in #1695 , that's failing here

**Review by stevepiercy — APPROVED** [review]
LGTM! Thank you!

**Comment by stevepiercy** [comment]
I also updated the branch from main. Let's merge when it all passes.

**Pull request merged**